fedimint-client/src/lib.rs

```diff
@@ -1943,6 +1944,7 @@ impl ClientBuilder {
     async fn migrate_database(&self, db: &Database) -> anyhow::Result<()> {
         // Only apply the client database migrations if the database has been
         // initialized.
+        // This only works as long as you don't change the client config
```

Comment:
Nit: this comment can be removed since we're applying the core client migrations before this